src/Workspaces/CoreTest/WorkspaceServiceTests/TemporaryStorageServiceTests.cs

```diff
@@ -98,10 +98,8 @@ public void TestZeroLengthStreams()
                 handle = service.WriteToTemporaryStorage(stream1, CancellationToken.None);
             }
```

Comment:
>} [](http://example.com/codeflow?start=12&length=1)

this too? or maybe there is a class of usings for which it can't determine the safety of a scope change?

Reply:
That would change semantics, moving where the Dispose is called during program execution

Reply:
that's a smart fixer!